src/main/java/Duke.java

```diff
@@ -6,5 +8,99 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        System.out.println("\t____________________________________________________________");
+        System.out.println("\tHello! I'm Duke");
+        System.out.println("\tWhat can I do for you?");
+        System.out.println("\t____________________________________________________________");
+        taskTracker();
+        System.out.println("\tBye. Hope to see you again soon!");
+        System.out.println("\t____________________________________________________________");
+    }
+
+    public static void taskTracker() {
```

Comment:
I like that this method is self-contained and that the logic flows nicely. Maybe the only thing would be that it is quite long, so maybe more abstraction can be done to separate the code into methods that are then called here (SLAP hard)